examples/dff/tests/Makefile

```diff
@@ -15,13 +15,17 @@ else
   $(error "A valid value (verilog or vhdl) was not provided for TOPLEVEL_LANG=$(TOPLEVEL_LANG)")
 endif
 
-TOPLEVEL=dff
-MODULE=$(TOPLEVEL)_cocotb
+TOPLEVEL = dff
+MODULE := $(TOPLEVEL)_cocotb
 
 CUSTOM_SIM_DEPS=$(CWD)/Makefile
 
 ifeq ($(SIM),questa)
-SIM_ARGS=-t 1ps
+    SIM_ARGS=-t 1ps
```

Comment:
The same problem is visible in Icarus (see above). Can't we add a "timescale" option to the file directly?

Reply:
Done in #1132.